### ISSUE TITLE
Package is missing icons?

### ISSUE THREAD

**Issue by philharmonie** [blade-ui-kit/blade-heroicons#39]
- Library Version: 1.4.0
- Laravel Version: 10.13.5
- PHP Version: 8.2.8

### Description:
Just one example: Heroicons has a icon called `arrow-path`.
The package does not have this.

### Steps To Reproduce:
```
composer require blade-ui-kit/blade-heroicons
find vendor/blade-ui-kit/blade-heroicons -type f -name "*path*"
```

**Comment by driesvints** [comment]
It exists? https://github.com/blade-ui-kit/blade-heroicons/blob/main/resources/svg/m-arrow-path.svg

**Comment by philharmonie** [comment]
Did you try to reproduce the issue before you closed it? It does not exist in the package

**Comment by driesvints** [comment]
It just works for me:

```blade
<x-heroicon-m-arrow-path />
```

<img width="1920" alt="Screenshot 2023-07-27 at 13 49 48" src="https://github.com/blade-ui-kit/blade-heroicons/assets/594614/b0645724-232e-43e3-a2bf-86af1e146c3e">

**Comment by philharmonie** [comment]
Thank you for testing. Any idea why some icons could not be downloaded? Locally this icon does not exist.

**Comment by driesvints** [comment]
I'm not sure.. it should just come with packagist.

**Comment by driesvints** [comment]
You might have an older version?

**Comment by driesvints** [comment]
Ah sorry, I now see you reported this as v1. The package is at v2 which should have this icon.

**Comment by philharmonie** [comment]
That's it! I just used the version that shows up on the right in the repo. Using ^2 fixed it!

**Comment by driesvints** [comment]
Ah that's a good spot. I fixed that to be 2.1.0 now. Thanks!
